### ISSUE TITLE
Cannot start due to panick in wgpu_core.rs

### ISSUE THREAD

**Issue by trongrg** [tim-harding/neophyte#25]
I want to try this editor, so I installed it on Mac (M3) using `cargo install neophyte`

The installed version is 0.3.2

when start the editor, it crash with error:

```
thread 'main' panicked at /Users/trongtran/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-23.0.1/src/backend/wgpu_core.rs:2976:18:
wgpu error: Validation Error

Caused by:
  In RenderPass::end
    In a pass parameter
      Attempted to use Texture with 'Monochrome texture' label (mips 0..1 layers 0..1) with conflicting usages. Current usage TextureUses(RESOURCE) and new usage TextureUses(COLOR_TARGET). TextureUses(COLOR_TARGET) is an exclusive usage and cannot be used with any other usages within the usage scope (renderpass or compute dispatch).
```

If you need the full backtrace, please let me know.

**Comment by tim-harding** [comment]
Thanks for letting me know. I'll take a look when I get the chance.

**Comment by tim-harding** [comment]
Fixed in 36a65a8 and new crate version is published. Seems that I was binding a different texture between pipeline initialization and update. Thanks for the report.
